01-messaging.md

```diff
@@ -284,6 +296,9 @@ The receiving node:
     - MAY fail the connection.
   - if the feature vector does not set all known, transitive dependencies:
     - MUST fail the connection.
+  - if it has no `compression_algorithms` in common with the sending node:
+    - MUST NOT exchange messages that require compression (e.g. gossip queries).
+    - MAY fail the connection.
```

Comment:
Unity any implementation supports the compression, and these are some things that can be useful for the future. Hat do you think to change this in

```suggestion
    - MUST not fail the connection.
```

Any only after when some compression algorithms are supported or required by the communication size add "MAY"?

Reply:
I'm sorry, I don't understand your comment at all. Can you rephrase? I think your proposed change is invalid.

Reply:
I'm sorry @t-bast, I think that there was some typo (fixed now) that makes it difficult to read the comment on the PR, and maybe a not useful question at all.

>Can you rephrase? 

Sure, I'm sorry. With my question I was pointing out I was missing the reason for failing the connection here between nodes, in other words, I was missing the case where the connection can fail here. However, I think that the case what a node may fail a connection is when the feature bit is > 0, but there is no match between compression algorithms.

I read now with another viewpoint, and I think this is the reason when a node can fail the connection?

But anyway I misunderstood some things when I read the spec. I'm so sorry.

Reply:
Yes, that seems to be it. Great, I'll resolve this comment then!